CONTRIBUTING.md

```
@@ -0,0 +1,67 @@
+
+---
+
+## 📜 Pull Request Guidelines
+
+1. **Create a Feature Branch:**
+   ```bash
+   git checkout -b feature/your-feature-name
+   ```
+2. **Quality Checks:**
+   Run lint and test commands before submitting:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)